src/System.Private.CoreLib/src/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -175,11 +175,6 @@ private unsafe int IndexOfCore(string source, string target, int startIndex, int
                 return IndexOfOrdinal(source, target, startIndex, count, ignoreCase: false);
             }
 
-            if (_isAsciiEqualityOrdinal && CanUseAsciiOrdinalForOptions(options) && source.IsFastSort() && target.IsFastSort())
```

Comment:
I guess we should keep the exact code we have in coreclr under \mscorlib\corefx\System\Globalization. right?